bigbluebutton-html5/imports/ui/components/error-screen/styles.scss

```diff
@@ -19,7 +19,35 @@
 }
 
 .message {
-  text-transform: uppercase;
+  margin: 0;
+  text-transform: capitalize;
+  color: var(--color-gray-light);
   font-size: 1.25rem;
-  font-weight: 600;
+  font-weight: 400;
 }
+
+.sessionMessage {
+  @extend .message;
+  font-size: var(--font-size-small);
+  margin-bottom: 1.5rem;
+}
+
+.codeError {
+  margin: 0;
+  font-size: 5rem;
+  color: var(--color-white);
+}
+
+.separator {
+  height: 0;
+  width: 5rem;
+  border: 1px solid var(--color-gray-lighter);
+  margin: 1.5rem 0 1.5rem 0;
+  align-self: center;
+  opacity: .75;
+}
+
+.button {
```

Comment:
![screenshot from 2019-01-11 11-53-30](https://user-images.githubusercontent.com/6312397/51047909-aa204f80-1597-11e9-9c38-2dc4c93a51dc.png)
Mobile views show the button distorted. In some cases I could not see the full text.